rodan-main/code/rodan/jobs/heuristic_pitch_finding/PitchFinding.py

```diff
@@ -497,10 +497,16 @@ def __glyph_type(g):
                 note = SCALE[int((clef_line - my_strt_pos + noteShift) % len(SCALE))]
 
                 # find octave
+                clef_octave_map = {
+                    'c': 4,
+                    'f': 3,
+                    'g': 5
```

Comment:
I don't think G5 is correct? I feel like a "default" g-clef is on G4.

Reply:
@kyrieb-ekat mentioned that in most cases it's G5. But I'm not sure, maybe she can confirm this. We can definitely switch it to G4 if needed.

Reply:
Should be G4.
https://en.wikipedia.org/wiki/Clef#G-clefs

Reply:
I'm skeptical... G5 isn't even in the gamut

Reply:
Not sure what I was thinking there, it should definitely be 4, that's on me. 🫡